themes/openy_themes/openy_rose/scss/paragraphs/_paragraphs.scss

```diff
@@ -0,0 +1,753 @@
+    height: 100%;
+    background: inherit;
+    content: '';
+    -webkit-transform: skew(20deg);
+    -moz-transform: skew(20deg);
+    -ms-transform: skew(20deg);
+    transform: skew(20deg);
+    @include breakpoint(0 $tablet) {
+      display: none;
+    }
+  }
+  .header-copy {
+    z-index: 3;
+    background: inherit;
+    .banner-copy {
+      padding: 30px 0;
+      z-index: 3;
+      h1 {
+        color: white;
+        margin-top: 0;
+        @include breakpoint(0 $tablet-start) {
+          font-size: 48px;
+        }
+      }
+      p {
+        font-size: 24px;
+        @include cachet();
+      }
+    }
+  }
+  .header-image {
+    z-index: 1;
+    height: 100%;
+    position: absolute;
+    left: 50%;
+    padding-right: 30px;
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    @include breakpoint($desktop) {
+      padding-right: 100px;
+    }
+    @include breakpoint(0 $tablet) {
+      background-image: none !important;
+      position: static;
+      width: 100%;
+      padding: 0;
+    }
+    figure {
+      overflow: hidden;
+      margin: 0 -30px 0 -20px;
+    }
+  }
+}
+
+.paragraph--type--promo-text {
+  padding: 20px 40px;
+  background: #e5e5e5;
+  border: 0;
+  position: relative;
+  font-weight: 200;
+  line-height: 1.4;
+  font-size: 21px;
+  z-index: 8;
+  &:before {
+    position: absolute;
+    top: 0;
+    display: block;
+    width: 80%;
+    height: 100%;
+    background-image: -webkit-linear-gradient(rgba(255, 255, 255, 0.43), rgba(255, 255, 255, 0));
+    background-image: -moz-linear-gradient(rgba(255, 255, 255, 0.43), rgba(255, 255, 255, 0));
+    background-image: -ms-linear-gradient(rgba(255, 255, 255, 0.43), rgba(255, 255, 255, 0));
+    background-image: linear-gradient(rgba(255, 255, 255, 0.43), rgba(255, 255, 255, 0));
+    content: '';
+    left: 45%;
+    -webkit-transform: skew(30deg);
+    -moz-transform: skew(30deg);
+    -ms-transform: skew(30deg);
+    -o-transform: skew(30deg);
+    transform: skew(30deg);
+  }
+  h2 {
+    color: #5e5e5e;
+    font-weight: bold;
+    font-size: 34px;
+    @include cachet();
+    text-transform: uppercase;
+    @include breakpoint (0 $tablet-start) {
+      font-size: 28px;
+    }
+  }
+  p {
+    font-size: 21px;
+  }
+  a {
+    font-weight: bold;
+  }
+  ul {
+    padding: 0;
+    li {
+      position: relative;
+      margin: 5px 0;
+      font-size: 21px;
+      padding-left: 20px;
+      list-style: none;
+      &:before {
+        position: absolute;
+        top: 2px;
+        left: 0;
+        margin-right: 8px;
+        color: #bebcbc;
+        content: '>';
+        font-family: 'YMCA';
+      }
+      a {
+        font-weight: bold;
+        @include cachet();
+      }
+    }
+  }
+}
+
+.paragraph--type--faq-item {
+  margin-top: 10px;
+  margin-bottom: 5px;
+  background-color: #fff;
+  border: 1px solid #a3a4a6;
+  border-radius: 4px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, .05) 0 0 #000;
```

Comment:
lets use either ready classes for shadows https://getbootstrap.com/docs/4.1/utilities/shadows/ or mixins

Reply:
rose theme.